src/xcode/ENA/ENA/Source/Scenes/Home/HomeViewController.swift

```diff
@@ -97,6 +97,13 @@ final class HomeViewController: UIViewController {
 		super.viewWillDisappear(animated)
 		NotificationCenter.default.removeObserver(summaryNotificationObserver as Any, name: .didDetectExposureDetectionSummary, object: nil)
 	}
+	
+	override func viewDidLayoutSubviews() {
+		let image = UIImage(named: "navi_bar_icon")
+		let leftItem = UIBarButtonItem(image: image, style: .plain, target: nil, action: nil)
+		leftItem.isEnabled = false
+		self.navigationItem.leftBarButtonItem = leftItem
+	}
```

Comment:
why not do it traitCollectionDidChange?

Reply:
Actually this should be handle by using appearances. There is no code required to change an image based on the current interface style.

Reply:
Also tried that implementation, but I tested both methods for calls that are unrelated to appearance changes (dark mode), and from what I've seen 'traitCollectionDidChange' was called a little bit more often than 'viewDidLayoutSubviews'. But sure both is possible without any difference. Actually Apple mentions both methods (and many more) as possibilities to place code like that.

So If you think that 'traitCollectionDidChange' is a better place for that kind of stuff, I will change that!

Reply:
Why does it have to be called in any of those two methods, anyway? Shouldn't that be a one-off configuration in `viewDidLoad`. Am I missing something?

Reply:
> Why does it have to be called in any of those two methods, anyway? Shouldn't that be a one-off configuration in `viewDidLoad`. Am I missing something?
@EisvogelSAP 
Thanks for your comment! As per my understanding `viewDidLoad()` is only called once when the controller view is loaded into the memory. It is possible to initialize the navigationItem in this method, but if the appearance changes when the app ist running, the navigationItem can't be re-loaded, so it would stuck on the ‚old‘ appearance. So practically you have to restart the app every time you change the appearance, to see the change happen on the navigationItem, because of the restricted one-time calling.

Reply:
Hi @tens0rfl0w,
Which method got called more offer, I think, it depends on what the experiment did. If you have manipulate the view hierarchy very often,  the `viewDidLayoutSubviews` will got called more often than `traitCollectionDidChange`. For our project, we did a lot of view manipulation on the home screen. So I think `traitCollectionDidChange` is better. Thanks.

Reply:
@haosap Fixed in the newest commit, please have a look!